cmake/BundledOpenCV.cmake

```diff
@@ -210,5 +210,7 @@ function(use_bundled_opencv SOURCE_DIR BINARY_DIR)
 
     add_library(OPENCV::libopencv INTERFACE IMPORTED)
     target_link_libraries(OPENCV::libopencv INTERFACE OPENCV::libopencv-flann OPENCV::libopencv-dnn OPENCV::libopencv-objdetect OPENCV::libopencv-core OPENCV::libopencv-gapi OPENCV::libopencv-imgcodecs OPENCV::libopencv-calib3d OPENCV::libopencv-imgproc OPENCV::libopencv-photo OPENCV::libopencv-videoio OPENCV::libopencv-video OPENCV::libopencv-stitching OPENCV::libopencv-features2d)
-
+    if (APPLE)
+        target_link_libraries(OPENCV::libopencv INTERFACE "-framework AVFoundation" "-framework CoreFoundation" "-framework CoreGraphics" "-framework CoreMedia" "-framework CoreVideo" "-framework Foundation" "-framework OpenCL" "-framework Accelerate")
```

Comment:
What's the docs reference that suggests this list of frameworks to be llnked? I wonder why it's not handled in the opencv cmake scripts.

Reply:
Afaik OpenCV handles this, I think we need this manual work due to our convoluted way of integrating opencv, I think we should be able to switch to fetchcontent but it wasnt trivial. 
I'll create a new Jira and investigate how painful that would be.